models.py

```diff
@@ -0,0 +1,26 @@
+from sqlalchemy import Column, Integer, String, Text, ForeignKey, Date
```

Comment:
There is a syntax error in the import statement. The correct way to import `Date` from `sqlalchemy` is without the slash. It should be `from sqlalchemy import Column, Integer, String, Text, ForeignKey, Date`.